src/test/java/ru/training/at/hw1/TestMultiply.java

```diff
@@ -0,0 +1,27 @@
+package ru.training.at.hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class TestMultiply {
+    private Calculator calculator = new Calculator();
+
+    @BeforeMethod
+    public void beforeMethod(){
+        calculator = new Calculator();
+    }
```

Comment:
Looks like that you've got in each test class `@BeforeMethod`, `@AfterMethod` and `Calculator` field. 
Could you please extract them to the base test class and **extends** all test classes from it